install.sh

```diff
@@ -347,6 +352,32 @@ get_install_dir() {
     printf %s "$_dir"
 }
 
+# Retrieve the version from the command-line arguments
+# Accepts flag: --version 1.1.0, --version v1.1.0, --version latest
+# Default: latest
+get_version() {
+    local _version
+    _version="latest"
+
+    while [[ $# -gt 0 ]]; do
+        case "$1" in
+        --version)
+            _version="$2"
+            shift 2
+            ;;
+        *)
```

Comment:
This and the loop in `get_install_dir` should be written the same way. I like this way better but either way, be consistent

Reply:
updated